packages/server/modules/workspaces/repositories/workspaces.ts

```diff
@@ -35,9 +42,32 @@ import { WorkspaceInviteResourceType } from '@/modules/workspaces/domain/constan
 const tables = {
   streams: (db: Knex) => db<StreamRecord>('streams'),
   workspaces: (db: Knex) => db<Workspace>('workspaces'),
+  workspaceDomains: (db: Knex) => db<WorkspaceDomain>('workspace_domains'),
   workspacesAcl: (db: Knex) => db<WorkspaceAcl>('workspace_acl')
 }
 
+export const getUserDiscoverableWorkspacesFactory =
+  ({ db }: { db: Knex }): GetUserDiscoverableWorkspaces =>
+  async ({ domains, userId }) => {
+    if (domains.length === 0) {
+      return []
+    }
+    return (await tables
+      .workspaces(db)
+      .select('workspaces.id as id', 'name', 'description')
```

Comment:
Yes, but you should use better judgement to not fuck up the column names. We've discussed how that only brings some value and no type safety, so that DX is arguable and subjective. I'm not a big fan of it, cause it makes things less readable. So lets not force it everywhere at every cost.

Reply:
> Yes, but you should use better judgement to not fuck up the column names.
Misspellings or forgetting to rename a changed string constant are not a "better judgement" question, its an easy thing to do for everyone.

The only downside I see is that its subjectively less readable to someone (seems just as readable to me, never had any issues with that), and other than that there's only positives.